### DISCUSSION TITLE
Network not working

### DISCUSSION THREAD

**Discussion by igorasilveira** [vercel/next.js#15113] [Help]
Hi, I am new to NextJS and Vercel and was following the official [tutorial](https://nextjs.org/learn/basics/create-nextjs-app?utm_source=next-site&utm_medium=homepage-cta&utm_campaign=next-website).

At the **deployment** lessons I followed all the steps and my GitHub repo is linked to the project within the Vercel dashboard, however, none of the provided urls seems to be serving content only  my own custom domain.

At the end, my code is just the same as the [final version](https://github.com/vercel/next-learn-starter/tree/master/typescript-final) of the tutorial (in the _typescript-final_ branch) so no actual change was made.

The urls of the project are:

- [nextjs-blog-git-master.dailydev.vercel.app](https://nextjs-blog-git-master.dailydev.vercel.app)
- [nextjs-blog.dailydev.vercel.app](https://nextjs-blog.dailydev.vercel.app)
- [nextjs-blog-delta-five.vercel.app](https://nextjs-blog-delta-five.vercel.app)
- [nextjs-blog-4mr0219h4.vercel.app](https://nextjs-blog-4mr0219h4.vercel.app)

Any extra configuration for the **.app** urls?
Anything I missed?

Thank you in advance 🙏

**Comment by paulogdm** [comment]
Hello Igor,

Can you run a few commands on your terminal?
```
dig A nextjs-blog-git-master.dailydev.vercel.app
dig NS vercel.app +short
dig A nextjs-blog-git-master.dailydev.vercel.app +short @1.1.1.1
```

Let us know.

**Reply by igorasilveira** [reply]
Olá Paulo, 😉

I'll list the results per command.

### dig A nextjs-blog-git-master.dailydev.vercel.app

```
; <<>> DiG 9.10.6 <<>> A nextjs-blog-git-master.dailydev.vercel.app
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 38771
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;nextjs-blog-git-master.dailydev.vercel.app. IN A

;; ANSWER SECTION:
nextjs-blog-git-master.dailydev.vercel.app. 59 IN A 76.76.21.21

;; Query time: 57 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Tue Jul 14 08:28:02 WEST 2020
;; MSG SIZE  rcvd: 87
```

### dig NS vercel.app +short

```
ns1.vercel-dns.com.
ns2.vercel-dns.com.
```

### dig A nextjs-blog-git-master.dailydev.vercel.app +short @1.1.1.1

```
76.76.21.21
```

I believe everything looks normal, but only my custom domain is responding.

Obrigado.
